src/main/kotlin/no/nav/soknad/arkivering/soknadsfillager/repository/FilDbData.kt

```diff
@@ -1,13 +1,14 @@
 package no.nav.soknad.arkivering.soknadsfillager.repository
 
+import java.time.LocalDateTime
 import javax.persistence.Column
 import javax.persistence.Entity
 import javax.persistence.Id
 import javax.persistence.Table
 
 @Entity
 @Table(name = "documents")
-data class FilDbData(@Id @Column(name = "id") val uuid: String, @Column(name = "data", columnDefinition = "bytea") val data: ByteArray?) {
+data class FilDbData(@Id @Column(name = "id") val uuid: String, @Column(name = "data", columnDefinition = "bytea") val data: ByteArray?, @Column(name = "opprettet", columnDefinition = "TIMESTAMP WITH TIME ZONE") val opprettet: LocalDateTime?) {
```

Comment:
burde det ikke het fil og ikke data?

Reply:
Enig

Reply:
For å endre kolonnenavn har jeg implementert migreringsskript også. 
Det som jeg ikke forstår i implementasjonen av FilDbData er equals metoden. Det kan da ikke være riktig. Jeg tror at eneste kriterium for likhet er at uuid er lik, ikke at det er samme fil.